mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -3,6 +3,10 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 		METHOD accept (Lnet/minecraft/class_1887;I)V
 			ARG 1 enchantment
 			ARG 2 level
+	METHOD method_17884 (Ljava/util/Map;Lnet/minecraft/class_2487;Lnet/minecraft/class_1887;)V
```

Comment:
Is this writing to the compound tag or what?

Reply:
It's writing to the map according to the compound tag.